action.yml

```diff
@@ -58,8 +67,16 @@ runs:
       with:
         endpoint: mycontext
 
+    - name: Validate skip scan
+      id: scan_validation
+      shell: bash
+      run: |
+        SKIP_SCAN=$(docker inspect ${{ inputs.image-tag }} | jq -r '.[0].Config.Labels | has("com.sysdig.ignore-failed-scan")')
```

Comment:
if we are making the opt-out from a scanning error by using the docker label (which is cool) we should remove this https://github.com/draios/infra-action-sysdig-scan/blob/5afb211ccd5c334ffa14ab93903caec79a07f8bf/action.yml#L28 or make it possible in both ways: from an action argument and a docker label.